### PR TITLE
fix paging problem

### PR DESCRIPTION
**Pull request by chuenlok**
item structure will lose after click the paging more than once

### CODE REVIEW

File: wagn/features/step_definitions/wagn_steps.rb [hunk 1]
```diff
@@ -342,6 +348,12 @@ def signed_in_as username
   page.has_css?(".diff-#{css_class}", text: text)
 end
 
+Then /^I should see css class "([^\"]*)" within "(.*)"$/ do |css_class, selector|
```
Comment (chuenlok) [inline wagn/features/step_definitions/wagn_steps.rb]: Can we exempt the line restriction in cucumber? It seems regular expression does not support multi-lines. I am not sure.

File: card/mod/05_standard/set/all/rich_html/wrapper.rb [hunk 2]
```diff
@@ -10,7 +10,11 @@ def slot_options args
     options_hash[:subslot] = 'true' if args[:subslot]
 
     @@slot_option_keys.inject(options_hash) do |hash, opt|
-      hash[opt] = args[opt] if args[opt].present?
+      if args[opt].present?
+        hash[opt] = args[opt]
+      elsif Env.params['slot'].present? && Env.params['slot'][opt.to_s].present?
+        hash[opt] = Env.params['slot'][opt.to_s]
+      end
```
Comment (ethn) [inline card/mod/05_standard/set/all/rich_html/wrapper.rb]: I think this will make the params override every slot on the page.

Comment (chuenlok) [inline card/mod/05_standard/set/all/rich_html/wrapper.rb]: How about adding a checking like
```ruby
elsif card.id == Card[Env.params['id']].id && Env.params['slot'].present? && Env.params['slot'][opt.to_s].present?
        hash[opt] = Env.params['slot'][opt.to_s]
      end
```

Comment (ethn) [inline card/mod/05_standard/set/all/rich_html/wrapper.rb]: that's making a mess.   The question is why the args aren't already there as they should be.

File: wagn/features/step_definitions/wagn_steps.rb (hunk 1 above)
Comment (ethn) [inline wagn/features/step_definitions/wagn_steps.rb]: could do.  could also just put regexp in a variable or variables in regexp...

File: card/mod/05_standard/set/all/rich_html/wrapper.rb (hunk 2 above)
Comment (ethn) [inline card/mod/05_standard/set/all/rich_html/wrapper.rb]: I've got a pr coming.  Your tests are a big help!!

**Pull request merged**